lib/rubocop/ast/processed_source.rb

```diff
@@ -306,6 +301,31 @@ def parser_class(ruby_version, parser_engine) # rubocop:disable Metrics/AbcSize,
         end
       end
 
+      # Prism is a native extension, a `LoadError` will be raised if linked to an incompatible
+      # Ruby version. Only raise if it really was caused by Prism not being present.
+      def require_prism
+        require 'prism'
+      rescue LoadError => e
+        raise unless e.path == 'prism'
+
+        warn "Error: Unable to load Prism. Add `gem 'prism'` to your Gemfile."
+        exit!
+      end
+
+      # While Prism is not yet a dependency, users may run with outdated versions that
+      # don't have all the parsers.
+      def require_prism_translation_parser(version)
+        require "prism/translation/parser#{version.to_s.delete('.')}"
+      rescue LoadError
+        warn <<~MESSAGE
+          Error: Unable to load Prism Parser for Ruby #{version}.
```

Comment:
That's just my two cents.

```suggestion
          Error: Unable to load Prism parser for Ruby #{version}.
```